docs/adr/adr_0001.adoc

```diff
@@ -16,59 +13,66 @@
+==== Question 1: Should We Unify the Concepts MetaDataProvider and PersistenceProvider?
 
-==== Question 1: Should we unify the concepts MetaDataProvider and PersistenceProvider?
-==== Solution approach 1: Unify
+===== Solution Approach 1: Unify
 
-Both "modules" are "processing" the security findings, which are generated in the parsing phase.
-But there is one larger difference between them:
+Both "`modules`" are "`processing`" the security findings, which were generated in the phase 2 "`parsing`",
+but there is one major difference between them:
 
-* `PersistenceProvider` is proccesing the findings with a *ReadOnly* pattern
-* `MetaDataProvider` is proccesing the findings with a *ReadAndWrite* pattern
+* a _PersistenceProvider_ is processing the findings *read only*, and
+* a _MetaDataProvider_ is processing the findings *read and write*.
 
-There is a similar thing in kubernetes called [AdmissionController](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/), with the exception that the are executed before a resource gets created.
+There is a similar concept in Kubernetes called https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/[AdmissionController], but with the exception that the will be executed before a resource is created.
 
-The are two variants:
+There are two variants of _AdmissionControllers_:
 
-* `ValidatingWebhookConfiguration` (ReadOnly) *Executed Last*
-* `MutatingWebhookConfiguration` (ReadAndWrite) *Executed First*
+. _ValidatingWebhookConfiguration_: *read only*, *executed last*; and
+. _MutatingWebhookConfiguration_: *read and write*, *executed first*.
 
-We could do a similar thing and introduce CRD which allows to execute "custom code" (generally speaking, depends on the second question) after a scan has completed (meaning both scan and parser phases are done). Some name ideas:
+We could do a similar thing and introduce CRD which allows to execute "`custom code`" (depends on the second question) after a scan has completed (meaning both phases "`scan`" and "`parsing`" were done). Some name ideas:
 
-- `ScanHooks`
-- `ScanCompletionHooks`
-- `FindingProcessors`
+* _ScanHooks_
+* _ScanCompletionHooks_
+* _FindingProcessors_
 
-These could be implemented with a `type` attribute, which declares if they are `ReadOnly` or `ReadAndWrite`.
+These could be implemented with a `type` attribute, which declares if they are *read only* or *read and write*.
 
-The operator would process all these resources in the namespace and execute the `ReadAndWrite` ones first (in serial: one at a time to avoid write conflicts) and then the `ReadOnly` ones (in parallel).
+The _secureCodeBox operator_ would process all these CRDs in the namespace and execute the *read and write* ones first in serial only one at a time to avoid write conflicts and then the *read only* ones in parallel.
+////
+#30: What namespace is meant here?
```

Comment:
What NS is meant here?

Reply:
kubernetes namespace: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/

Reply:
"in the" implies that it is a special one. Is it the namespace of the operator or which one?

Reply:
Ah yeah good note. "In the current namespace" would be better.

Reply:
Or "in the kubernetes namespace of the Scan"

Reply:
Done